.github/workflows/workflow.yml

```diff
@@ -0,0 +1,35 @@
+
+# Copyright (c) 2022 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+name: Mirror Container Images
+
+on:
+  push:
+    branches: [ main ]
+  schedule:
+    - cron: 0 5 * * *
```

Comment:
this will run every 5am instead of midnight. not sure which one you intended but the pr description says midnight

Reply:
The GitHub actions uses UTC, so it's 5AM UTC, which is midnight EST 🙂